<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicCutView</class>
 <widget class="QWidget" name="MusicCutView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnOpenFile">
         <property name="text">
          <string>打开文件</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leInFileName">
         <property name="text">
          <string>./local/123.mp3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnOpenOutputPath">
         <property name="text">
          <string>打开文件夹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leOutputPath">
         <property name="text">
          <string>./local</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>输出文件名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leOutputFilename">
         <property name="text">
          <string>jianji.mp3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>开始时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leStartTime">
           <property name="text">
            <string>20.123</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>结束时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leEndTime">
           <property name="text">
            <string>20.123</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="btnStart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
